containers/ecr-viewer/src/app/services/userService.ts

```diff
@@ -204,6 +198,33 @@ const createUserQuery = async (
   return uuid;
 };
 
+/**
+ *
+ * @param db the database connection
+ * @param email email of the user to update (case-insensitive)
+ * @param uuid uuid of the user to update
+ * @throws UserFacingError when anothe user with the same email exists
+ */
+const checkDupeEmail = async (
+  db: Transaction<Core>,
+  email: string | null | undefined,
+  uuid: string,
+) => {
+  if (!email) {
+    return;
+  }
+
+  const user = await getUserByEmail(email, db);
+  // user does not exist or is this user
+  if (!user || user.uuid === uuid) {
+    return;
+  }
+
+  throw new UserFacingError(
+    "This email already exists. Please add a different email.",
```

Comment:
```suggestion
    "This email already exists. Please use a different email.",
```

just to be a bit more generic since this shows on the update vs create page

Reply:
I made this change before I saw this comment. I went with `Another user with this email already exists.` as an analog to programAreaService `This program area name already exists.` I don't mind using your suggestion though.